docs/connecting-to-a-lit-network/migrating-to-datil.md

```diff
@@ -11,19 +11,19 @@ Depending on the existing Lit network you are using, you should migrate to its c
 
 | Currently Available | Requires Payment | Minimum Lit Package Version | Your Current Network | Network to Migrate to | Description                                                  |
 |---------------------|------------------|-----------------------------|----------------------|-----------------------|--------------------------------------------------------------|
-| ❌                   | ✅                | n/a                         | `habanero`           | `datil-prod`               | Decentralized mainnet designed for production use cases      |
+| ✅                   | ✅                | `6.4.0`                     | `habanero`           | `datil`               | Decentralized mainnet designed for production use cases      |
 | ✅                   | ✅                | `6.2.2`                     | `manzano`            | `datil-test`          | Decentralized testnet designed for pre-production deployment |
 | ✅                   | ❌                | `6.2.2`                     | `cayenne`            | `datil-dev`           | Centralized testnet designed for early-stage development     |
 
-Like their counterparts, `datil-prod` and `datil-test` require developers to pay for usage of the Lit network via [Capacity Credits](../capacity-credits); however, `datil-dev` does not.
+Like their counterparts, `datil` and `datil-test` require developers to pay for usage of the Lit network via [Capacity Credits](../capacity-credits); however, `datil-dev` does not.
 
 ## Breaking Changes and Important Updates
 
 - Chronicle Yellowstone's chain facts are available [here](./lit-blockchains/chronicle-yellowstone.md#connecting-to-chronicle-yellowstone).
 
 ### New Network, New PKPs
 
-PKPs minted on the existing Lit networks: `cayenne`, `manzano`, and `habanero` exist on the Chronicle blockchain. Because of this, when migrating to the new Datil networks: `datil-dev`, `datil-test`, and `datil-prod`, your PKPs will need to be re-minted on the Chronicle Yellowstone blockchain. This also means transferring ownership of assets owned by PKPs minted on Chronicle, to the newly minted ones on Chronicle Yellowstone.
+PKPs minted on the existing Lit networks: `cayenne`, `manzano`, and `habanero` exist on the Chronicle blockchain. Because of this, when migrating to the new Datil networks: `datil-dev`, `datil-test`, and `datil`, your PKPs will need to be re-minted on the Chronicle Yellowstone blockchain. This also means transferring ownership of assets owned by PKPs minted on Chronicle, to the newly minted ones on Chronicle Yellowstone.
```

Comment:
Maybe say "...on earlier Lit networks" rather than "on the existing Lit networks"

Reply:
Fixed